src/generator/operations.ts

```diff
@@ -627,6 +665,9 @@ function createInterfaceDefinition(group: OperationGroup, imports: ImportManager
       // don't generate a public API for the methods used to advance pages
       continue;
     }
+    if (op.language.go!.methodPrefix) {
+      op.language.go!.name = `${op.language.go!.methodPrefix}${op.language.go!.name}`;
```

Comment:
This will change the underlying name which is different than what we did elsewhere, is this safe?

Reply:
I agree. I was thinking about this but didn't see it change anything else. Not safe or guaranteed however. I'll change it.

Reply:
Done.